include/aws/common/common.h

```diff
@@ -144,7 +144,10 @@ AWS_COMMON_API void aws_load_error_strings(void);
 typedef enum aws_common_error {
     AWS_ERROR_SUCCESS = 0,
     AWS_ERROR_OOM,
+    AWS_ERROR_BAD_FREE,
```

Comment:
I don't see where you've added these strings to the errors in common.c

Reply:
Thanks, good catch!